packages/admin/dashboard/src/i18n/translations/ru.json

```diff
@@ -0,0 +1,2752 @@
+{
```

Comment:
Some new keys were added, so would be good to get them in:

```
Validation failed for ru.json:
  Missing required key: "transfer" at orders
  Missing required key: "from" at orders.activity
  Missing required key: "to" at orders.activity
  Missing required key: "transfer" at orders.activity.events
  ```

Reply:
I'll do! Thank you!